### ISSUE TITLE
Intégrer la librairie Y-POJO et comprendre réellement ce que cela fait et pourquoi Tom l'a écrite.

### ISSUE THREAD

**Issue by micbuffa** [Doori4N/musical-multiverse-vr#7]
Tom, auteur de sequencer.party a écrit cette librairie spécialement pour synchroniser les wams. Etudier le code, et relire son mail pour bien comprendre ce qu'il raconte et pourquoi il a fait cela 👍 
-------

So the issue, of course, is that the WAM state is a plain old javascript object (a POJO), and to synchronize that state object you need to put it into a YJS document.  I built a library to do this for you.

It was a pain in the ass to get right.

Note that I keep every WAM in it's own YJS document. 

Updating YJS documents will get slower and slower as you append more edit events onto them so if you keep your objects segmented into separate YJS documents you will delay that inevitable growth. Also between sessions I do not store the YJS document, I only store the JSON representation of all objects and then create new YJS documents in the next session. This can be tricky to determine which client should create the YJS document if there's more than one client.  I had the server do it to avoid that problem.

So basically what you need to do is have a function that runs every 100ms, say, and have it do wam.getState() and then use the synchronize method (from the y-pojo library) to map any WAM changes onto the YJS document.  I'd assume the client already has a callback registered with YJS to send YJS updates over the network.

Then on the receiving side, when the YJS document holding the WAM state receives an update you have to flatten the YJS document to JSON and then call wam.setState().

It also has code to dynamically filter out any WAM fields that are connected to modulations to prevent a modulated parameter from spamming the network with updates.
